src/server.c

```diff
@@ -426,45 +426,42 @@ int dictExpandAllowed(size_t moreMem, double usedRatio) {
  * 
  * In non-cluster mode, bucket count can be retrieved directly from single dict bucket and
  * we don't need this list as there is only one dictionary per DB. */
-void dictRehashingStarted(dict *d) {
+void dictRehashingStartedByKeyType(dict *d, dbKeyType keyType) {
```

Comment:
The processing methods of dict and expire dict are exactly the same.
If this cleaning is not important, i can also drop it.